src/Faker/Provider/th_TH/Person.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Faker\Provider\th_TH;
+
+class Person extends \Faker\Provider\Person
+{
+    /**
+     * @var array Thai male first name
+     * @link http://www.amarinbabyandkids.com/app/childname/
+     */
+    protected static $firstNameMale = array(
+        'กชฐณณ', 'กชนนท์', 'กชมนณัฐ', 'กนกภพ', 'กมลวร', 'กรณ์วรรธน์', 'กรภณ', 'กรภัทร', 'กรฤต', 'กรวรรธ', 'กช', 'กฤตพรต',
```

Comment:
There is not enough data in this class for a good randomization. can you find a datasource with a larger data set?

Reply:
Got your point. But I've got no idea how many items would be good, any suggestions?

Reply:
about 300 entries